.nav.yml

```diff
@@ -5,3 +5,4 @@ nav:
   - 'Chain Interactions': chain-interactions
   # 'Get Support': get-support
   - 'Nodes and Validators': nodes-and-validators
+  - 'Technical Reference': reference
```

Comment:
```suggestion
```

This just needs to be removed for now, will be addressing this in a separate PR.